src/views/ProposalDetail.vue

```diff
@@ -0,0 +1,343 @@
+<template>
```

Comment:
This template was pulled over from the backend... I split up what I pulled over into a few different components to keep this component less complicated.